_posts/2017-10-18-long-term-survey-results.md

```diff
@@ -0,0 +1,67 @@
++ *Productivity*: My research productivity has improved as a result of completing the workshop.
++ *Motivation*: I have been motivated to seek more knowledge about the tools I learned at the workshop.
++ *Confidence*: I have gained confidence in working with data as a result of completing the workshop.
++ *Coding*: I have improved my coding practices as a result of completing the workshop.
++ *Career*: I have used skills I learned at the workshop to advance my career.
+
+![alt text](https://github.com/carpentries/assessment/blob/master/carpentries/long-term-survey/figures/workshop_impact_heatmap.png) 
+
+The figure shows that respondents agree or strongly agree about gaining confidence in working with data (85.3%), making their analyses more reproducible (69.4%), and receiving professional recognition for their work (64.7%) all as a result of attending a Software or Data Carpentry workshop.
+
+From this figure we also see that there are opportunities to improve. Motivation to seek more knowledge seems unchanging, likely because learners who attend our workshops are already motivated. Perhaps these learners remain enthusiastic, having high pre-workshop motivation scores, rather than a decrease in motivation post-workshop. We’d also like to see a shift in positive trend for learners using the skills they learned to advance their career, which is why we are implementing round two of the [Carpentries Mentoring Program](http://www.datacarpentry.org/blog/mentoring-round-2/) this fall.
+
+Interested in reading more? The full report is available, and it provides more detailed information about the motivation behind this survey, respondent demographics, and growth opportunities. Take a look at the [report](https://carpentries.github.io/assessment-projects/joint-carpentry-projects/long-term-survey/report.html). We will continue to conduct this assessment at 6 month intervals to capture feedback from people who took workshops 6 months or more ago. Additionally, assessment will be the focus of our October community call. Bring your thoughts to the [community call](http://pad.software-carpentry.org/community-call-2017-10-19) October 19th!
+The surveys used in this work, anonmyized data and R scripts for generating the figures are available in our [assessment repository](https://carpentries.github.io/assessment/carpentries/long-term-survey/). This report was made possible by community input from [Ben Marwick](https://github.com/benmarwick), [Belinda Weaver](https://github.com/weaverbel), [Naupaka Zimmerman](https://github.com/naupaka), [Jason Williams](https://github.com/JasonJWilliamsNY), [Tracy Teal](https://github.com/tracykteal), [Erin Becker](https://github.com/ErinBecker), [Jonah Duckles](https://github.com/jduckles), [Beth Duckles](https://github.com/bduckles), [Elizabeth Wickes](https://github.com/elliewix). We thank you all so much for your contributions to the code in this report and development of our long-term survey!
+
```

Comment:
The link to the report should be updated to https://carpentries.github.io/assessment/carpentries/long-term-survey/report.html